### DISCUSSION TITLE
The tailwind-light theme conflicts with the globals.css style of Tailwind.

### DISCUSSION THREAD

**Discussion by farhadibehnamdev** [primefaces/community#175] [PrimeReact] [answered]
Hi there, I'm working on an ecommerce website and admin panel with Next.js  and Tailwind CSS. After I found PrimeReact has good potential, I installed it and added the tailwind-light theme in my project as it conflicted with the globals.css style of Tailwind. I reproduced the conflict issue.

[https://replit.com/@BehnamFarhadi/primereacttest#pages/_app.tsx](https://replit.com/@BehnamFarhadi/primereacttest#pages/_app.tsx)

**Comment by habubey** [comment]
As far as I can tell, the tailwind setup has not been implemented following the instruction. Do not forget to wrap your app.
```
        <PrimeReactProvider value={{ unstyled: true, pt: Tailwind }}>
            <ThemeSwitcher />
             <App />
        </PrimeReactProvider>
```
Please check this page, https://primereact.org/tailwind/

**Reply by farhadibehnamdev** [reply]
I attempted `<PrimeReactProvider value={{ unstyled: true, pt: Tailwind }}>`, but it resulted in the loss of Primereact components' default styles. As my present project does not require custom styles (perhaps only minor modifications to the default ones), the default styles are already satisfactory for my needs.

**Reply by habubey** [reply]
What do you mean when you say default styles loss? When we say value={{ unstyled: true, pt: Tailwind }}, only Tailwind styles will appear.

**Reply by melloware** [reply, marked as answer]
@farhdibehnamdev your issue is this: https://github.com/primefaces/primereact/issues/4987  please subscribe and follow that ticket.
